opnode/test/setup.go

```diff
@@ -151,8 +151,9 @@ func initializeGenesis(cfg *systemConfig) {
 		Difficulty: common.Big1,
 		GasLimit:   5000000,
 		Nonce:      4660,
-		Timestamp:  genesisTimestamp,
-		BaseFee:    big.NewInt(7),
+		// Set the L2 timestamps to be before the L1 timestamps. This is to match the current implementation of timestamps.
```

Comment:
```suggestion
		// Set the L2 timestamps to be before the timestamp of the L1 genesis. This is to match the current implementation of timestamps.
```

Reply:
Think this is worth cleaning up later. The genesis L2 block is "derived" from the anchor L1 block I'd say, and equal genesis timestamps should be possible, just like it should be possible to match the timestamp of the L1 block you start a new epoch of L2 blocks with. Not a big deal in this test though, and can be separated into later timestamp work